tests/pydyad_spsc/run.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# FLUX: -N 2
+# FLUX: --output=pydyad_spsc_test.out
+# FLUX: --error=pydyad_spsc_test.err
+
+DYAD_INSTALL_LIBDIR="/g/g90/lumsden1/ws/insitu_benchmark/dyad/_test_install/lib"
+KVS_NAMESPACE="pydyad_test"
+CONS_MANAGED_PATH="/l/ssd/lumsden1/pydyad_test"
```

Comment:
Same with these variables. No hardcoded path to one of yours.

Reply:
These paths were hardcoded because this was mainly my script for testing on Corona. I've changed it to be environment variable-based